### PR TITLE
Update libraries page

### PR DESCRIPTION
**Pull request by tindzk**

### CODE REVIEW

File: _data/library/scalalibs.yml [hunk 1]
```diff
@@ -121,14 +125,14 @@
       url: https://github.com/lihaoyi/scala.rx
       desc: Change-propagation/FRP library
       dep: '"com.lihaoyi" %%% "scalarx" % "0.2.8"'
-    - name: Monifu
-      url: https://github.com/alexandru/monifu
+    - name: Monix
+      url: https://github.com/monixio/monix
       desc: Reactive extensions (Rx) with back-pressure, atomic references and other multi-threading primitives
-      dep: '"org.monifu" %%% "monifu" % "1.0-RC3"'
+      dep: '"io.monix" %% "monix" % "2.0-RC5"'
```
Comment (sjrd) [inline _data/library/scalalibs.yml]: This should keep the `%%%`.

**Comment by sjrd** [comment]
That's all.

**Comment by tindzk** [comment]
Indeed, thanks! I fixed it.

**Comment by sjrd** [comment]
LGTM, thanks!

**Pull request merged**